From: Caleb <yocalebo@gmail.com>
Date: Sat, 4 Apr 2020 12:14:26 -0400
Subject: flake8 fixes for gfs.metrics utilization

---
 gfs/metrics/utilization.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gfs/metrics/utilization.py b/gfs/metrics/utilization.py
index 08dfb93..93b7ec7 100644
--- a/gfs/metrics/utilization.py
+++ b/gfs/metrics/utilization.py
@@ -9,7 +9,7 @@ def local_utilization(volname=None):
 
     :param volname: Volume Name
     :returns: List of utilization information
-        {               
+        {
             "volume": VOLUME_NAME,
             "brick_index": BRICK_INDEX_IN_VOL_INFO,
             "node_id": NODE_ID,
