From: Caleb <yocalebo@gmail.com>
Date: Sat, 4 Apr 2020 12:16:57 -0400
Subject: flake8 fixes for gfs.cli peer

---
 gfs/cli/peer.py | 7 ++++---
 1 file changed, 4 insertions(+), 3 deletions(-)

diff --git a/gfs/cli/peer.py b/gfs/cli/peer.py
index 96eb4ff..7e6afdc 100644
--- a/gfs/cli/peer.py
+++ b/gfs/cli/peer.py
@@ -61,20 +61,21 @@ def detach_all():
         for peer in peers:
             host = peer["hostname"]
             if peer["connected"] == "Connected":
-                cmd = ["detach",host]
+                cmd = ["detach", host]
                 try:
                     result = peer_execute(cmd)
-                    out = str(host)+" "+result
+                    out = str(host) + " " + result
                     outlist.append(out)
                 except Exception as err:
                     errors_list.append(err)
             else:
-                err = str(host)+" is not connected"
+                err = str(host) + " is not connected"
                 errors_list.append(err)
     if len(errors_list):
         raise GlusterCmdException((1, "", errors_list))
     return "/n".join(outlist)
 
+
 def status():
     """
     Peer Status of Cluster
